react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,26 @@
-import { isArray } from './utils';
+import isArray from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+const returnOneArray = (array) => array.reduce((acc, val) => acc.concat(val), []);
+
+function min(...params) {
+  if (params.length === 0) return undefined;
+
+  return isArray(...params)
```

Comment:
You don't have to check the whole params, just check the first position of params. This will let you know whether the arguments are in an array or in a list of numbers.